src/PKSim.Core/Model/ExpressionProfile.cs

```diff
@@ -45,22 +59,21 @@ public override string Name
                return;
 
             var names = CoreConstants.NamesFromCompositeName(value);
-            if (names.Count != 2)
+            if (names.Count != 3)
                return;
 
             _moleculeName = names[0];
-            _category = names[1];
+            _category = names[2];
```

Comment:
yes. There is a specifc presneter for this
![image](https://user-images.githubusercontent.com/1041237/142241924-f4285015-3977-48fd-b347-605a822dd775.png)

Reply:
That's good. However I still don't understand why we have the Name setter.
`Name.get` should just return {Protein}-{Species}-{Phenotype}.
Especially now if I e.g. set Protein name = "P-gp" or Phenotype="Disease - stage 1" it will not work, because the value is splitted by "-".